Testing/Unit/sitkCommonTests.cxx

```diff
@@ -739,6 +739,8 @@ TEST( Interpolator, Test1 )
   ss.str("");
   ss << sitk::sitkBlackmanWindowedSinc;
   EXPECT_EQ("BlackmanWindowedSinc", ss.str());
+  ss << sitk::sitkLabelLinear;
```

Comment:
Missing `ss.str("")'

Test failure:
https://open.cdash.org/test/1211771265
[ RUN      ] Interpolator.Test1
/home/circleci/SimpleITK/Testing/Unit/sitkCommonTests.cxx:743: Failure
Expected equality of these values:
  "LabelLinear"
  ss.str()
    Which is: "BlackmanWindowedSincLabelLinear"

Reply:
thanks, I was still trying to setup a proper build (not SuperBuild), but I cmake doesn't like my lua executable.